README.md

```diff
@@ -27,6 +27,16 @@ User-agent: anthropic-ai
 # Common Crawl
 User-agent: CCBot
 
+# Omglibot: webz.io
+User-agent: Omgilibot
```

Comment:
A lot of robots.txt restricts those user agents, but I don't think this is specific to LLM crawlers.

I would argue against this one, and consider GPTBot as a better marker for this list.

At least, we should add a comment to explain this is a general-purpose crawler.

WDYT?

Reply:
Webz.io advertises itself as a source of high quality data for model training: https://webz.io/datasets/premium-datasets

But you may be right. The list of crawlers used to train AI models is open-ended. If I had a big dataset, I'd try to monetize it now, during the hype stage :smile: